internetarchive/__init__.py

```diff
@@ -79,13 +77,6 @@
 
 # Set default logging handler to avoid "No handler found" warnings.
 import logging
-try:  # Python 2.7+
-    from logging import NullHandler
-except ImportError:
-    class NullHandler(logging.Handler):
-        def emit(self, record):
-            pass
-
 
 log = logging.getLogger(__name__)
-log.addHandler(NullHandler())
+log.addHandler(logging.NullHandler())
```

Comment:
In my eyes `from logging import NullHandler` and then using that name instead of the fully qualified name would be better.

Reply:
The original import for Python versions >= 2.7 was `from logging import NullHandler`, `import logging` was there as a fallback, so I would keep the former.

Reply:
The `import logging` was and is also used by `logging.getLogger`.